content/applications/finance/accounting/reporting/data_inalterability.rst

```diff
@@ -20,22 +20,31 @@ would break the hash chain.
 
 .. note::
    Hashes generated by the SHA-256 algorithm are theoretically not unique, as there is a finite
-   number of possible values. However, this number is exceptionally high: 2²⁵⁶, which is a lot
-   bigger than the number of atoms in the known universe. This is why hashes are considered unique
-   in practice.
+   number of possible values. However, this number is exceptionally high: 2 :sup:`256`, which is a
+   lot bigger than the number of atoms in the known universe. This is why hashes are considered
+   unique in practice.
```

Comment:
Actually I modified it because it looked weird on my browser, but yeah perhaps it's not worth breaking translations :)

Before

![image](https://github.com/odoo/documentation/assets/32981518/30f2a60d-7c57-4553-a8be-2136d3f05eca)


After
![image](https://github.com/odoo/documentation/assets/32981518/c2ac851e-a991-4dc5-8f6d-5ee87128cf33)

Reply:
oh weird.

Reply:
In any case, we'd fix this with another PR so it can be applied to all versions